app/javascript/controllers/comments-modal_controller.js

```diff
@@ -0,0 +1,24 @@
+import { Controller } from "stimulus";
+
+export default class extends Controller {
+    static targets = ['id', 'comments'];
+
+    setCoHostContent(data) {
+        this.idTarget.innerHTML = data.id;
+        this.commentsTarget.innerHTML = data.comments.replace(/\n/g,'<br />');
+    }
+
+    open() {
+        document.body.classList.add("modal-open");
+        this.element.setAttribute("style", "display: block;");
+        this.element.classList.add("show");
+        document.body.innerHTML += '<div id="modal-backdrop" class="modal-backdrop fade show"></div>';
```

Comment:
stylistic choice but to me its a bit easier to go back and make changes this was done via creating an element, setting its ID and classes and then appending modifying the HTML in code

Reply:
I'm not sure I understand what you are saying for this one

Reply:
something like this 

`let modalBackdrop = document.createElement('div')
modalBackdrop.classList.add("modal-backdrop");
modalBackdrop.classList.add("fade");
modalBackdrop.classList.add("show");
modalBackdrop.id = "modal-backdrop"
document.body.append(modalBackdrop)`

instead of 
`document.body.innerHTML += '<div id="modal-backdrop" class="modal-backdrop fade show"></div>'; `

I find it more readable and easier to make changes to